src/core/lib/transport/handshaker.cc

```diff
@@ -58,16 +58,20 @@ std::string HandshakerArgsString(HandshakerArgs* args) {
 
 }  // namespace
 
-HandshakeManager::HandshakeManager() {}
+HandshakeManager::HandshakeManager()
+    : RefCounted(GRPC_TRACE_FLAG_ENABLED(grpc_handshaker_trace)
+                     ? "HandshakeManager"
+                     : nullptr),
+      event_engine_(GetDefaultEventEngine()) {}
```

Comment:
Using `GetDefaultEventEngine()` is a great start to prove the migration will work! I did a quick investigation, and I think ultimately (before we merge the PR) the right thing to do here may be to pass an `std::shared_ptr<EventEngine>` argument in the Handshakemanager constructor. The call sites that construct a HandshakeManager will likely have an instance available in their channel_args.

Reply:
Thanks for the hint! This really seems to work!